doc/nrf/config_and_build/bootloaders/bootloader_adding_sysbuild.rst

```diff
@@ -144,8 +144,8 @@ See :ref:`ug_fw_update_keys` for information on how to generate custom keys for
 
 Additionally, the |NSIB| supports the following methods for signing images with private keys:
 
-* :ref:`ug_fw_update_keys_openssl` - Uses the :kconfig:option:`SB_CONFIG_SECURE_BOOT_SIGNING_OPENSSL` Kconfig option.
-* :ref:`Using a custom command <ug_bootloader_adding_sysbuild_immutable_b0_custom_signing>` - Uses the :kconfig:option:`SB_CONFIG_SECURE_BOOT_SIGNING_CUSTOM` Kconfig option.
+* :ref:`ug_fw_update_keys_openssl` - Uses the ``SB_CONFIG_SECURE_BOOT_SIGNING_OPENSSL`` Kconfig option.
```

Comment:
This links to `Using OpenSSL to generate keys`. Do we actually want to link there here? (section describes signing images)

Reply:
@nordicjm Can you check this?

Reply:
@nvlsianpu, could you take a look